dvc/scm/git/__init__.py

```diff
@@ -18,6 +19,13 @@
 logger = logging.getLogger(__name__)
 
 
+class TqdmGit(Tqdm):
+    def update_git(self, op_code, cur_count, max_count=None, message=""):
```

Comment:
Hm, don't we need an `op_code`? It tells us which clone stage we are on.

Reply:
atm I'm not parsing it. Could either:

1. ignore it
2. prefix current `message` with a formatted version of `op_code`

Reply:
reference: https://gitpython.readthedocs.io/en/stable/reference.html?highlight=RemoteProgress#git.remote.RemoteProgress.update


- op_code -
    Integer allowing to be compared against Operation IDs and stage IDs.

    Stage IDs are BEGIN and END. BEGIN will only be set once for each Operation ID as well as END. It may be that BEGIN and END are set at once in case only one progress message was emitted due to the speed of the operation. Between BEGIN and END, none of these flags will be set

    Operation IDs are all held within the OP_MASK. Only one Operation ID will be active per call.

-    cur_count – Current absolute count of items

-    max_count – The maximum count of items we expect. It may be None in case there is no maximum number of items or if it is (yet) unknown.

-    message – In case of the ‘WRITING’ operation, it contains the amount of bytes transferred. It may possibly be used for other purposes as well.

Reply:
decided to parse `op_code` & update description accordingly, and use `message` in the postfix instead.